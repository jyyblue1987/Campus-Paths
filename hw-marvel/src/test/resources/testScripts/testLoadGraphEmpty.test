# Test load a graph with no nodes

LoadGraph empty emptyFile.csv

FindPath empty n1 n2