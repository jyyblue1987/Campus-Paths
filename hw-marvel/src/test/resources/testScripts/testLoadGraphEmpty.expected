# Test load a graph with no nodes

loaded graph empty

unknown: n1
unknown: n2